Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -179,6 +188,58 @@ public Task<bool> IsContestIpValidByContestAndIp(int contestId, string ip)
                 isUserAdmin);
         }
 
+        public async Task<ContestValidationModel> ValidateContestNew(Contest contest, string userId, bool isUserAdmin,
```

Comment:
Method should be renamed

Reply:
Created a validation service that has a method GetValidationResult. The logic of ValidateContestNew is moved there.